### PR TITLE
checkpoint-04

### PR DESCRIPTION
**Pull request by jkoo87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,11 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+def offer_rose(person)
```
Comment (juancgarcia) [inline README.md:32]: 👍

File: README.md [hunk 2]
```diff
@@ -54,7 +58,8 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+town[:residents].delete("Belle")
```
Comment (juancgarcia) [inline README.md:61]: 👍

File: README.md [hunk 3]
```diff
@@ -77,7 +82,9 @@ Belle is friends with Mrs. Potts
 Write your code here...
 
 ```ruby
-# code here
+friends.each do |friend|
```
Comment (juancgarcia) [inline README.md:85]: 👍

File: README.md [hunk 4]
```diff
@@ -100,7 +107,35 @@ Each lion should have...
...
+
+  def initialize(name)
+    @name = name
+  end
+
+  def greet
+    puts "My name is #{@name}"
+  end
+end
+
```
Comment (juancgarcia) [inline README.md:121]: Missed creating the pumba instance:
```ruby
pumba = Animal.new("Pumba")
```

File: README.md [hunk 5]
```diff
@@ -118,7 +153,7 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+ERD is an Entity-Relationship Diagram which gives an overview of your models and how they are associated. For example, 'Genie' is related to 'Lamp'(one-to-one), and 'Pet' is related to 'Person'(many-to-many).
```
Comment (juancgarcia) [inline README.md:156]: 👍

File: README.md [hunk 6]
```diff
@@ -129,7 +164,8 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+Each schema defines what each columns it has in application's database.
```
Comment (juancgarcia) [inline README.md:167]: We wanted you to talk about how we can show a relationship between tables. For instance how a foreign key like `person_id` might be used to relate the wish table to the `people` table.

**Review by juancgarcia — COMMENTED** [review]
5.5/8

Question 7, and 8 were unanswered. Did you perhaps forget to push your last commit? If so, let me know and I can review those questions and update this score.

**Comment by jkoo87** [comment]
I actually did not have time to complete all of the questions on that day..

File: README.md (hunk 6 above)
Comment (jkoo87) [reply README.md:167]: I actually did not have time to complete all of the questions on that day..

**Pull request closed without merging**